pages/upgrade-guide-2.x.md

```diff
@@ -196,6 +196,36 @@ QUnit.module( "router", {
 ```
 
 
+## Modified assertion methods
+
+A few assertion methods have been modified.
```

Comment:
Can do, but I don't think this fits in any of the existing sections. Or do
you mean the subheading should be promoted to h2?
On Jul 3, 2016 2:19 PM, "Trent Willis" <notifications@github.com> wrote:

> In pages/upgrade-guide-2.x.md
> <https://github.com/jquery/qunitjs.com/pull/123#discussion_r69396120>:
>
> > @@ -196,6 +196,36 @@ QUnit.module( "router", {
> >  ```
> >
> >
> > +## Modified assertion methods
> > +
> > +A few assertion methods have been modified.
>
> Since there's only one thing listed here, can we ditch this modified
> assertion methods heading? Doesn't seem beneficial.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jquery/qunitjs.com/pull/123/files/732b9bceed5c086dc0eccf2c93c8aa778efb64c7#r69396120>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AARWejTA4xNCEXzs9QEMeWUcpFFye6aeks5qSAtDgaJpZM4JD5ct>
> .
>